pkg/api/v1/conversion.go

```diff
@@ -125,7 +125,9 @@ func addConversionFuncs() {
 	err = api.Scheme.AddFieldLabelConversionFunc("v1", "Namespace",
 		func(label, value string) (string, string, error) {
 			switch label {
-			case "status.phase":
+			case "name":
```

Comment:
Ok I removed those labels. But they are still present in respective `SelectableFields()` ([Namespace](https://github.com/miminar/kubernetes/blob/namespace_field_conv/pkg/registry/namespace/strategy.go#L151), [PersistentVolume](https://github.com/miminar/kubernetes/blob/namespace_field_conv/pkg/registry/persistentvolume/strategy.go#L110), [PersistentVolumeClaim](https://github.com/miminar/kubernetes/blob/namespace_field_conv/pkg/registry/persistentvolumeclaim/strategy.go#L110)). And without the conversion labels they aren't usable.